app/assets/javascripts/application/sticky_tools.js

```diff
@@ -55,6 +57,3 @@ $(function() {
     scroller();
   }
 });
-
```

Comment:
I replaced it with `//= require bootstrap-sass/bootstrap/affix`. This seems to be just the minified version of that file inlined for some reason.